apps/avatax/src/modules/avatax/avatax-client.ts

```diff
@@ -76,14 +76,7 @@ export class AvataxClient {
             model: { createTransactionModel: model },
           }),
           (error) => {
-            const parsedError = this.errorParser.parse(error);
-
-            /**
-             * TODO: Refactor errors so we are able to print error only for unhandled cases, otherwise use warnings etc
```

Comment:
If we have unhandled case - we send exception to the Sentry. In case of handled case we log this error in route as warn.